Makefile

```diff
@@ -75,7 +75,7 @@ standalone_chrome: generate_standalone_chrome chrome
 generate_standalone_chrome_debug:
 	cd ./StandaloneDebug && ./generate.sh StandaloneChromeDebug node-chrome-debug Chrome $(VERSION) $(NAMESPACE) $(AUTHORS)
 
-standalone_chrome_debug: generate_standalone_chrome_debug standalone_chrome
+standalone_chrome_debug: chrome_debug generate_standalone_chrome_debug standalone_chrome
```

Comment:
I think you just need to replace `standalone_chrome` with `chrome_debug`, instead of adding it. 
Could you please change that?

I also noted that the `standalone_firefox_debug` has the same issue, if you want to, make that change as well.

Reply:
I'll merge it like that and then make the additional change. Thanks again!